### ISSUE TITLE
`cursorOverStyle` adapter

### ISSUE THREAD

**Issue by dmsmaxim** [amcharts/amcharts4#610]
I don't know why, but there is no adapter for `cursorOverStyle` on many elements.
Because of that, there is no ability to change the cursor by some condition...
It would be nice to have the ability to write something like this:
```
pieSeries.slices.template.adapter.add('cursorOverStyle', argum => 
   condition(argum) ? am4core.MouseCursorStyle.pointer : am4core.MouseCursorStyle.default);
```

**Comment by martynasma** [comment]
Agreed. We could use an adapter for that. Except not directly on `cursorOverStyle` which is a shortcut to `cursorOptions.overStyle`, but for whole `cursorOptions`.

We'll assess implementing this.

Will keep you posted here.

Thanks for the suggestion!

**Comment by martynasma** [comment]
Fixed in [4.0.8](https://github.com/amcharts/amcharts4/releases/tag/4.0.8).

## [4.0.8] - 2018-12-11

### Added
- `currentStep` added to `ColorSet`. Allows setting current position of the color iterator.
- Adapters added for `Sprite` properties: `inertiaOptions`, `hitOptions`, `hoverOptions`, `swipeOptions`, `keyboardOptions`, `cursorOptions`.

### Changed
- `nonScaling` of `AxisLabel` is no longer set to `true` by default, to improve performance.

### Fixed
- `ValueAxis` was not updating its scale if series was hidden and animated theme was not enabled.
- Zooming an `XYChart` with all values equal was broken.
- Series in an `XYChartScrollbar` sometimes was not visible initially.
- `ColumnSeries` was drawing 1px line for `null` value data items.
- `DateAxis`' `snapTooltip = true` was working incorrectly with `skipEmptyPeriods = true`.
- Logarithmic `ValueAxis` with axis breaks was resulting in stack overflows.
- JSON: Chaining multiple adapters in an array (e.g. `adapter: [{...}, {...}]`), was not working.
- JSON: Adding elements to `children` via array syntax was not working properly.
- JSON: `Container` elements can now be created via JSON config.
- Setting `dataFields` in `Export` was only affecting column names. Values for all data fields were still being included in export.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
